### PR TITLE
initial bootstrap

### PR DESCRIPTION
**Pull request by grokspawn**

### CODE REVIEW

File: ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    from: src
+- as: vet
+  commands: make vet
+  container:
+    from: src
+- as: fmt
+  commands: make fmt
+  container:
+    from: src
+zz_generated_metadata:
```
Comment (ncdc) [inline ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml:58]: Please add the verify-deps target à la https://github.com/openshift/release/blob/a029cc215f57d17a9e38bb30ec427127cc042067/ci-operator/config/openshift/operator-framework-catalogd/openshift-operator-framework-catalogd-main.yaml#L64

Comment (grokspawn) [reply ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml:66]: Roger that.  I was going to wait until I had the existing tests running, to iterate through the cross-fittings, but I've added it now.

**Comment by grokspawn** [comment]
/pj-rehearse pull-ci-openshift-operator-framework-operator-controller-main-images

**Comment by stevekuznetsov** [comment]
/pj-rehearse pull-ci-openshift-operator-framework-operator-controller-main-images

**Review by ncdc — APPROVED** [review]

**Review by ncdc — APPROVED** [review]

File: ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.19-openshift-4.14:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.14
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |-
```
Comment (bparees) [inline ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml:13]: why isn't the dockerfile in the repo?

File: core-services/image-mirroring/openshift/mapping_origin_4_14 [hunk 3]
```diff
@@ -237,6 +237,7 @@ registry.ci.openshift.org/origin/4.14:oc-mirror quay.io/openshift/origin-oc-mirr
 registry.ci.openshift.org/origin/4.14:oc-rpms quay.io/openshift/origin-oc-rpms:4.14 quay.io/openshift/origin-oc-rpms:4.14.0 quay.io/openshift/origin-oc-rpms:latest
 registry.ci.openshift.org/origin/4.14:okd-rpms quay.io/openshift/origin-okd-rpms:4.14 quay.io/openshift/origin-okd-rpms:4.14.0 quay.io/openshift/origin-okd-rpms:latest
 registry.ci.openshift.org/origin/4.14:olm-catalogd quay.io/openshift/origin-olm-catalogd:4.14 quay.io/openshift/origin-olm-catalogd:4.14.0 quay.io/openshift/origin-olm-catalogd:latest
+registry.ci.openshift.org/origin/4.14:olm-operator-controller quay.io/openshift/origin-olm-operator-controller:4.14 quay.io/openshift/origin-olm-operator-controller:4.14.0 quay.io/openshift/origin-olm-operator-controller:latest
```
Comment (bparees) [inline core-services/image-mirroring/openshift/mapping_origin_4_14:240]: proposed image name seems fine

File: ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml (hunk 2 above)
Comment (ncdc) [reply ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml:13]: If we are diverging from the majority, we'll switch to in-repo

Comment (bparees) [reply ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml:13]: yeah a quick grep shows me a handful of `dockerfile_literal` usage, but not a ton(relatively speaking) and i don't think any of them are in the payload

**Pull request merged**